src/firebase.js

```diff
@@ -0,0 +1,60 @@
+import firebase from "firebase";
+import 'firebase/firestore';
+
+require("dotenv").config();
+
+// Set collection name
+const collectionName = "db_pilot_test";
+
+// Firebase config
+let config = {
+  apiKey: process.env.REACT_APP_apiKey,
+  authDomain: process.env.REACT_APP_authDomain,
+  databaseURL: process.env.REACT_APP_databaseURL,
+  projectId: process.env.REACT_APP_projectId || "no-firebase",
+  storageBucket: process.env.REACT_APP_storageBucket,
+  messagingSenderId: process.env.REACT_APP_messagingSenderId,
+  appId: process.env.REACT_APP_appId,
+};
+
+
+
+// Get a Firestore instance
+var db = firebase.initializeApp(config).firestore();
+
```

Comment:
hmmmm

Reply:
what have you tried for debugging this so far? What is the error you see?

Reply:
This should be solved. I started a new project by cloning this branch and the emulator works. I think it may have been something with the specific project instance on my end.